samples/core/ai-platform/README.md

```diff
@@ -27,10 +27,10 @@ The [Chicago Crime Data](https://console.cloud.google.com/marketplace/details/ci
 ## Assets
 
 * Notebooks
-  * [Chicago Crime Research.ipynb](Chicago%20Crime%20Research.ipynb) Illustrates the process of building a **model** outside of production.
-  * [Chicago Crime Pipeline.ipynb](Chicago%20Crime%20Pipeline.ipynb) Illustrates how to build a **pipeline** and deploy it to a Kubeflow instance
+  * [Chicago Crime Pipeline.ipynb](Chicago%20Crime%20Pipeline.ipynb) Illustrates how to build a **pipeline** and deploy it to a Kubeflow instance.
+  * [/local/Chicago Crime Research.ipynb](Chicago%20Crime%20Research.ipynb) Illustrates the process of building a **model** outside of production.
```

Comment:
Those links are not actually working. Do we want to keep them anyway or to use the code style?

Reply:
Let me remove the links. Good catch.

Reply:
Done.